### DISCUSSION TITLE
Can I plot TRS using Violinplot instead?

### DISCUSSION THREAD

**Discussion by dmsalsgh97** [sankaranlab/SCAVENGE#7] [General]
Hi, thanks for developing the SCAVENGE. 
I'm trying to use SCAVENGE on my trait-tissue pair, and it works quite well :)

Now, I want to plot TRS scores, and I noticed that on your paper, boxplot is used instead of violin plot.

When I plot my results as a boxplot, It seems like below.
![image](https://user-images.githubusercontent.com/42495757/181170522-5e04eb6d-bb20-4ef5-ad2e-2ba3409b0365.png)

And the same result looks like below when I used violin plot instead.
![image](https://user-images.githubusercontent.com/42495757/181170912-bdcc2226-2048-4c5f-b9d0-98f78135aa16.png)

Using the violinplot, I found that 3rd, and 4th cluster has some highely scored cells(TRS >3) and It is reasonable because those are known as the most important cells in the trait. 

So, in this case, can I use a violin plot instead?
thanks.

**Comment by fl-yu** [comment]
Thank you for using our tool!

It looks great and is definietly fesible to use violin plot instead!
